### PR TITLE
Move PricingCard and FeatureCard from index.tsx to new component files

### PR DESCRIPTION
**Pull request by hummus-erectus**
Move PricingCard and FeatureCard from index.tsx to new component files

### CODE REVIEW

File: app/index.tsx [hunk 1]
```diff
@@ -4,6 +4,9 @@ import { useRouter } from 'expo-router';
 import { ArrowRight, Mic, Book, BarChart, Users, Play } from 'lucide-react-native';
 import { LinearGradient } from 'expo-linear-gradient';
 
+import PricingCard from '@/components/PricingCard';
```
Comment (jjmountain) [inline app/index.tsx:7]: Do you think we could store these in components/landing_page as it's likely we'll only ever use them there?

Comment (hummus-erectus) [reply app/index.tsx:7]: Yes that sounds better

**Comment by jjmountain** [comment]
@hummus-erectus you didn't need to close this pull request 😢 

Here's what you should do (and still could if you re-open it)

- make sure you are on the branch locally (`git checkout Move-components-to-files`)
- make the change
- add and commit the changes
- push the branch
- the pull request will be updated and I'll be able to approve

That's the typical workflow. Of course it's very normal for people to request changes 😀

**Comment by hummus-erectus** [comment]
Gomen! I'm slowly getting it. I ended up making some more commits and opening a seperate pull request. I'll know better for next time, thanks for correcting me.

**Pull request closed without merging**